### DISCUSSION TITLE
Roadmap for Orkes Conductor Community server

### DISCUSSION THREAD

**Discussion by oxsdd** [orkes-io/orkes-conductor-community#13] [Q&A] [answered]
I am evaluating the various Conductor server packages available (Netflix, Community, Orkes Community) for potential use on-prem.

The version of this package  is 1.0.3 and the README states that it is fully compatible with Netflix Conductor, and that Postgres is used for indexing enabling full text search. However, only limited search capabilities work in the UI, and appears that some functionality is not implemented in the code. It appears that this package is still a work-in-progress. Can you clarify its status and future intentions?

What was the motivation for replacing Elasticsearch with Postgres for full-text and archiving? And if so, why is REDIS still used for persistence and queues, and not Postgres for everything?

Thanks!

**Comment by boney9** [comment, marked as answer]
Sorry - it looks like this discussion went unanswered  - leaving this here if you are still curious about the questions posted

- Re: search, for cost optimization we are using postgres here which is an lot cheaper than ES / Opensearch
- Redis for queues - makes it way more performant than just using postgres on its own
- We released this package and its already used in production by a few companies preferring OSS and we intend to fully support this more actively going forward - we also offer an enterprise edition on top of this so its naturally going to be maintained that way
- This is a prescriptive Postgres + Redis flavor of Conductor, but you can follow a similar model to assemble your own preferred combo too!
- The build here is quite light weight - the primary code base is the Netflix OSS repo and we pull those jars to build this one

**Comment by boney9** [comment]
(closing this - in case you have follow up questions, please let us know or join our slack)

**Comment by oxsdd** [comment]
Thanks for the update. At the time of writing we were just beginning to investigate options and the many possible configurations and relative positioning of Community and Orkes Community was not clear.  Also, when this was originally posted, it seemed that Orkes Community Server lacked indexing functionality required for full operation of OSS Conductor React UI, which we were also interested in using. We have been evaluating Community + Postgres (Execution, Queues, Indexing) since our apps already use Postgres as a DB. Using Postgres for queues is a slight concern, but we want to see if it will be sufficient before adding REDIS to the mix.
